packages/upstash-redis/tests/index.test.ts

```diff
@@ -0,0 +1,39 @@
+import upstashRedisClient from "@upstash/redis"
+import { runBasicTests } from "../../../basic-tests"
+import UpstashRedisAdapter from "../src"
+
+const client = upstashRedisClient(
+  process.env.NEXTAUTH_UPSTASH_REDIS_URL,
+  process.env.NEXTAUTH_UPSTASH_REDIS_KEY
+)
+
+const reviveFromJson = (json: string) =>
+  JSON.parse(json, (key, value) =>
+    key === "emailVerified" || key === "expires" ? new Date(value) : value
+  )
+
+runBasicTests({
+  adapter: UpstashRedisAdapter(client, { baseKeyPrefix: "testApp:" }),
+  db: {
+    verificationToken: async (where) => {
+      const { data } = await client.get(
+        `testApp:user:token:${where.identifier}`
+      )
+      return reviveFromJson(data)
+    },
+    user: async (id: string) => {
+      const { data } = await client.get(`testApp:user:${id}`)
```

Comment:
```suggestion
      const { data } = await client.get(`testApp:user:${id}`)
```

Why are `testApp` prefixes necessary, if you already added a `baseKeyPrefix` on line 16?